content/Developer_Program/WebAuthn_Starter_Kit/Adaptive_Multi-Factor_Authentication.adoc

```diff
@@ -0,0 +1,169 @@
+== Adaptive Multi-Factor Authentication
+
+=== Introduction
+
+The username and password combination has long been entrenched as the primary form of authentication for better or worse, and it has placed strict limits on the flexibility of user authentication flows. With the advent of WebAuthn, and as its adoption gains momentum as an authentication scheme, an opportunity to remove user friction without compromising security or being dependent on a federated identity arises. While account security can be entirely protected by WebAuthn, there are cases where additional methods of authentication should be considered due to a variety of factors such as:
+* The browser or platform being used by the user
+* The protocol of the users authenticator
+* The users recurrent use of a specific device
+ 
+Adaptive Multi-Factor Authentication should be considered as a way to provide the highest degree of account security with flexibility to ensure the proper amount of friction is presented to the user depending on the current context of their session. Many services offer Adaptive Multi-Factor Authentication using multiple authentication types, such as Security keys paired with smartphone apps. However, it is possible to offer a similar level of user ease of use paired with security with just the features included in WebAuthn.
```

Comment:
Consider explicitly stating how the term Adaptive MFA is interpreted by the community today in this paragraph (e.g. risk profile, geo-location, etc...).  We should set the context/background before we deep dive into the `with WebAuthn` modifier/vertical.